# zhuoyu1 zhuoyu2 zhuoyu3 haokong1 evo haokong2 
MOTOR :
  zhuoyu1 :
    SDO_ID : 0x601
    PDO_ID : 0x201
    Encoding_rate : 20

  zhuoyu2 :
    SDO_ID : 0x602
    PDO_ID : 0x202
    Encoding_rate : 20

  zhuoyu3 :
    SDO_ID : 0x603
    PDO_ID : 0x203
    Encoding_rate : 20

  haokong1 :
    SDO_ID : 0x604
    PDO_ID : 0x204
    Encoding_rate : 17    
  evo1 : 
    SDO_ID : 0x605
    PDO_ID : 0x205
    Encoding_rate : 17

  haokong2 :
    SDO_ID : 0x606
    PDO_ID : 0x206
    Encoding_rate : 17

zhuoyu4 :
    SDO_ID : 0x607
    PDO_ID : 0x207
    Encoding_rate : 20

  zhuoyu5 :
    SDO_ID : 0x608
    PDO_ID : 0x208
    Encoding_rate : 20

  zhuoyu6 :
    SDO_ID : 0x609
    PDO_ID : 0x209
    Encoding_rate : 20

  haokong3 :
    SDO_ID : 0x60A
    PDO_ID : 0x20A
    Encoding_rate : 17    
  evo2 : 
    SDO_ID : 0x60B
    PDO_ID : 0x20B
    Encoding_rate : 17

  haokong4 :
    SDO_ID : 0x60C
    PDO_ID : 0x20C
    Encoding_rate : 17



CAN :
  frame_SDO_interval : 5
  frame_PDO_interval : 5
  sync_interval : 30


